Gemfile.lock

```diff
@@ -341,7 +341,7 @@ GEM
       thor (>= 0.18.1, < 2.0)
     rainbow (2.2.2)
       rake
-    rake (12.0.0)
+    rake (11.3.0)
```

Comment:
yes, it depend on annotate_models.
https://github.com/ctran/annotate_models/blob/v2.7.1/annotate.gemspec#L32